### PR TITLE
suport cache template and recompile partials

### PR DESCRIPTION
**Pull request by daifee**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

1. suport cache template
2. optimize local dev(always recompile partials, when render page)

### CODE REVIEW

**Comment by daifee** [comment]
@popomore @atian25 等review

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,4 @@ coverage/
 run/
 .DS_Store
 *.swp
-
+#package-lock.json
```
Comment (atian25) [inline .gitignore:9]: remove `#`

File: README.md [hunk 2]
```diff
@@ -1,31 +1,24 @@
-# egg-view-handlebars
+# egg-view-handlebars-x
```
Comment (atian25) [inline README.md:1]: `x` ?

File: app.js [hunk 3]
```diff
@@ -5,35 +5,61 @@ const fs = require('mz/fs');
 const handlebars = require('handlebars');
 
 const COMPILE = Symbol('compile');
+const cacheStore = new Map();
```
Comment (atian25) [inline app.js:8]: use `LRU` ?

**Review by atian25 — CHANGES_REQUESTED** [review]

**Comment by daifee** [comment]
i have published egg-view-handlebars-x to npm

**Pull request closed without merging**